packages/node/example/bootstrappedApp/src/app.js

```diff
@@ -0,0 +1,43 @@
+const { Session } = require("../../../dist/Session");
+
+const argv = require("yargs/yargs")(process.argv.slice(2))
+  .describe("clientId", "The registered client ID.")
+  .alias("id", "clientId")
+  .describe("clientSecret", "The secret associated to the client ID.")
+  .alias("secret", "clientSecret")
+  .describe("refreshToken", "The API token (OAuth refresh token).")
+  .alias("token", "refreshToken")
```

Comment:
I'd just remove this alias completely, as `token` is so overloaded (i.e. ID token, Access token, Refresh token).

Reply:
Fixed in f8b6c127da465222224fdccc7dc07ca254567302